openedx/openedx-platform
# [ENT-208] Update AWS ENTERPRISE_ENROLLMENT_API_URL variable

We determined that when not explicitly set, `ENTERPRISE_ENROLLMENT_API_URL` will always default to being on `localhost:8000`, rather on the relevant host. This pull request updates the setting in the AWS file to ensure that in production, the URL used is dependent on the actual base URL.

**JIRA tickets**: Fixes [ENT-208](https://openedx.atlassian.net/browse/ENT-208)

**Dependencies**: None

**Sandbox URL**: https://pr14556.sandbox.opencraft.hosting (use staff/edx user account)

**Testing instructions** (must be tested on sandbox):

1. Create an EnterpriseCustomer in the sandbox admin
2. Add a variety of custom course modes for the demo course included in the sandbox
3. Go to the EnterpriseCustomer's Manage Learners admin panel
4. Enter a course ID to enroll a user in
5. Verify that the course modes you created are populated into the Mode dropdown, and that there are no errors in the Javascript console.

**Reviewers**

**Settings**
```yaml
EDXAPP_FEATURES:
  ENABLE_COMBINED_LOGIN_REGISTRATION: true
```
